### PR TITLE
Refactoring commands to manage endpoints, adding missing endpoint man…

### PR DESCRIPTION
**Pull request by anuchandy**
…agement features and adding commands to manage load balanced endpoint set

### CODE REVIEW

File: lib/commands/asm/service.lbset._js [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+//
+var util = require('util');
+var profile = require('../../util/profile');
+var utils = require('../../util/utils');
+var EndPointUtil = require('../../util/endpointUtil');
+
+var $ = utils.getLocaleString;
+
+exports.init = function(cli) {
+  var lbSet = cli.category('service').category('load-balancer-set')
```
Comment (anuchandy) [inline lib/commands/asm/service.lbset._js]: The command category name must be 'load-balanced-set' not 'load-balancer-set'. Also update all text 'load balancer' with 'load balanced'

File: lib/commands/asm/service.lbset._js [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+        deployment = computeClient.deployments.getBySlot(serviceName, 'Production', _);
+      } finally {
+        progress.end();
+      }
+
+      var endPointUtil = new EndPointUtil();
+      var lbSets = endPointUtil.getAllLBSettings(deployment.roles);
+      cli.interaction.formatOutput(lbSets, function(lbSets) {
+        if (lbSets.length === 0) {
+          log.info($('No load balancer sets defined'));
```
Comment (ruslangabitov) [inline lib/commands/asm/service.lbset._js]: Should we check if --json option was specified and print '{}'? Like we do in ARM?

File: lib/commands/asm/service.lbset._js [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+              row.cell($('Probe interval (sec)'), lbSet.ProbSettings.intervalInSeconds);
+              row.cell($('Probe timeout (sec)'), lbSet.ProbSettings.timeoutInSeconds);
+            }
+          });
+        }
+      });
+    });
+
+  lbSet.command('show [service-name] [load-balanced-set-name]')
+    .description($('List load balancer set for a cloud service deployment'))
```
Comment (ruslangabitov) [inline lib/commands/asm/service.lbset._js]: Looks like it must be 'Show load balancer ...'

File: lib/commands/asm/service.lbset._js [hunk 4]
```diff
@@ -0,0 +1,181 @@
...
+      var progress = cli.interaction.progress($('Getting cloud service deployment'));
+      var deployment;
+      try {
+        deployment = computeClient.deployments.getBySlot(serviceName, 'Production', _);
+      } finally {
+        progress.end();
+      }
+
+      var endPointUtil = new EndPointUtil();
+      var lbSets = endPointUtil.getAllLBSettings(deployment.roles);
```
Comment (ruslangabitov) [inline lib/commands/asm/service.lbset._js]: What happens if deployment was not found? Will 'deployment' variable be undefined and throw an error here?

File: lib/commands/asm/service.lbset._js [hunk 5]
```diff
@@ -0,0 +1,181 @@
...
+          output.json({});
+        } else {
+          output.warn(util.format($('A load balancer set with name "%s" not found in the service "%s"'), loadBalancedSetName, serviceName));
+        }
+      }
+    });
+
+
+  lbSet.command('set [service-name] [load-balanced-set-name]')
+    .description($('List load balancer set for a cloud service deployment'))
```
Comment (ruslangabitov) [inline lib/commands/asm/service.lbset._js]: Description from 'list' command

**Comment by ruslangabitov** [comment]
LGTM.

File: lib/commands/asm/service.lbset._js (hunk 4 above)
Comment (anuchandy) [inline lib/commands/asm/service.lbset._js]: The SDK will throw the error 'No deployments were found.' and cli command will end in this case.

File: lib/commands/asm/service.lbset._js (hunk 2 above)
Comment (anuchandy) [inline lib/commands/asm/service.lbset._js]: Thanks, we should. I will fix it,.

**Pull request merged**